src/controller/CommissioningDelegate.h

```diff
@@ -51,6 +51,10 @@ enum CommissioningStage : uint8_t
     kSendAttestationRequest,     ///< Send AttestationRequest (0x3E:0) command to the device
     kAttestationVerification,    ///< Verify AttestationResponse (0x3E:1) validity
     kAttestationRevocationCheck, ///< Verify Revocation Status of device's DAC chain
+#if CHIP_DEVICE_CONFIG_ENABLE_JOINT_FABRIC
+    kJFValidateNOC,              ///< Verify Admin NOC contains an Administrator CAT
+    kSendVIDVerificationRequest, ///< Send SignVIDVerificationRequest command to the device
+#endif                           // CHIP_DEVICE_CONFIG_ENABLE_JOINT_FABRIC
```

Comment:
We should probably put these at the end of the enum, so the numbering of the other bits does not depend on whether JF is enabled?

Reply:
Enum follows the transitions of the commissioner state machine so it will be a little bit confusing if I put those JF bits at the end - I can do it if you want to. Or maybe I can keep those value present even if CHIP_DEVICE_CONFIG_ENABLE_JOINT_FABRIC is not defined?

Reply:
Let's just keep the values present no matter what.  There is no cost to it, and it simplifies things a lot.

Reply:
Can we leave the cases in the switch even when JF is not defined, but just condition the actual logic on JF?

It's really important for client sanity to have stable values for this enumeration for all the other steps that don't depend on whether JF is enabled.